liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseRollbackOneChangeSetSQL.java

```diff
@@ -108,7 +108,8 @@ protected void performLiquibaseTask(Liquibase liquibase) throws LiquibaseExcepti
         boolean hasProLicense = MavenUtils.checkProLicense(liquibaseProLicenseKey, commandName, getLog());
         if (! hasProLicense) {
             throw new LiquibaseException(
-               "The command 'rollbackOneChangeSetSQL' requires a Liquibase Pro License, available at http://www.liquibase.org/download or sales@liquibase.com.");
+               "The command 'rollbackOneChangeSetSQL' requires a Liquibase Pro License, available at http://www.liquibase.org/download or sales@liquibase.com." +
```

Comment:
This code is just for maven, so the --pro-license-key part doesn't apply here. There is a different way we should be explaining how to set it up in maven